models/boost/arbitrum/boost_arbitrum_schema.yml

```diff
@@ -0,0 +1,82 @@
+version: 2
+
+models:
+  - name: boost_arbitrum_deployed
+    meta:
+      blockchain: arbitrum
+      project: boost
+      contributors: raho, pntemi
+    config:
+      tags: ["arbitrum", "boost", "boost_inbox", "quest"]
+    description: Boost deployments on [boost.xyz](https://boost.xyz), an open source platform to incentivize any onchain action, with any token, on any chain. 
```

Comment:
please also add the `unique_combo_of_columns` test in here + all other schema files which contains models with unique keys
you can find examples in the `docs/` directory or yml files throughout `_sector/` directory

Reply:
added to all spells that are incremental